src/applications/mhv/medications/components/shared/FillRefillButton.jsx

```diff
@@ -12,15 +12,11 @@ const FillRefillButton = rx => {
     dispensedDate,
     error,
     prescriptionId,
-    refillRemaining,
-    dispStatus,
     success,
+    isRefillable,
   } = rx;
 
-  if (
-    (dispStatus === 'Active' && refillRemaining !== 0) ||
-    dispStatus === 'Active: Parked'
-  ) {
+  if (isRefillable) {
```

Comment:
Not sure it is safe here. Please take a look at FillRefillButton change history (refer to Slack chat)